backend/yandex/yandex_crawl.py

```diff
@@ -0,0 +1,49 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+def to_dict(index, sauce, img_sauce):
+    table = {
+         "img_link": img_sauce,
+         "source": sauce,
+         "host": "Yandex",
+         "tittle": "yandex"+str(index)
+         }
+    return table
```

Comment:
Do we really need this function? If yes, please give it a docstring and use it.

Reply:
yeah, i forgot to delete it after applying Long ways